recipes/onnx/all/conandata.yml

```diff
@@ -1,4 +1,13 @@
 sources:
+  "1.13.1":
+    url: "https://github.com/onnx/onnx/archive/refs/tags/v1.13.1.tar.gz"
+    sha256: "090d3e10ec662a98a2a72f1bf053f793efc645824f0d4b779e0ce47468a0890e"
+  "1.13.0":
+    url: "https://github.com/onnx/onnx/archive/refs/tags/v1.13.0.tar.gz"
+    sha256: "66eb61fc0ff4b6189816eb8e4da52e1e6775a1c29f372cbd08b694aa5b4ca978"
+  "1.12.0":
+    url: "https://github.com/onnx/onnx/archive/refs/tags/v1.12.0.tar.gz"
+    sha256: "052ad3d5dad358a33606e0fc89483f8150bb0655c99b12a43aa58b5b7f0cc507"
```

Comment:
isn't it excessive? 1.13.1 is enough I think, maybe 1.12.0, but not 1.13.0

Reply:
1.13.0 is the exact version used in the latest onnxruntime (1.14.1) so if work is resumed on getting onnxruntime in cci, I'd rather have it around than not at first.
I will remove 1.12.0

Reply:
(comment not specific to this recipe) Many times I had to maintain a recipe copy because CCI recipe either dropped support for some version or the version i required was not in CCI. Wouldn't it be better to keep the most detailed conandata.yml as possible, but then control what CCI builds with config.yml? (Leaving the non-built references as 'best-effort')

Reply:
The problem is perception, its very difficult to communicate clearly that a recipe or config might work.

It's more frustrating to waste time on a dead end. If it's not there at least you know it's going to take effort.

You are always more then welcome to submit PRs and add older versions, it's document in the FAQ.

I am all ears if you have a ideas for the human problem :)

Reply:
Looking at https://github.com/onnx/onnx/releases/tag/v1.13.1 I don't understand why you need the exact version since there's bugs.

Is there a particular reason you don't want those fixes?

Reply:
In my case I don't need 1.13.1 but 1.12.0. And the reason is because in my organization we still have to mantain the legacy-build while we migrate to Conan (almost 2 years in the process and still not there).

Reply:
> Is there a particular reason you don't want those fixes?

> 1.13.0 is the exact version used in the latest onnxruntime (1.14.1) so if work is resumed on getting onnxruntime in cci, I'd rather have it around than not at first.

This was my only incentive to get 1.13.0, I understand that it's not a good enough reason.

From:
> 1.13.1 is enough I think, maybe 1.12.0, but not 1.13.0
And
> in my case I don't need 1.13.1 but 1.12.0.

I'll remove 1.13.0 and add 1.12.0 back in.